SpatialGDK/Documentation/content/handover-between-server-workers.md

```diff
@@ -19,20 +17,17 @@ Note that server-worker authority over properties is different to server-worker
 ## How to facilitate Actor handover
 
 To facilitate an Actor’s property handover between server-workers, follow the instructions below:
-1.  In the Actor’s class, mark the property field with a `Handover` tag in the `UPROPERTY` macro, as shown in the example below.
+1.  If your property is defined in a native c++ class, mark the property field with a `Handover` tag in the `UPROPERTY` macro, as shown in the example below.
 
 ```
 UPROPERTY(Handover)
 float MyServerSideVariable;
 ```
 
-2. In the Actor's `UCLASS` macro, mark the class as a `SpatialType`, as show in the example below.
+2. Alternatively, if your property is defined in a blueprint class, set the variable's replication setting to `Handover` within the Blueprint Editor. 
```

Comment:
This line starts with a "2." but should probably begin with "1." so that the automatic numbering can kick in (presuming the following lines are meant to be 3 and 4.

Reply:
I'd recommend doing a local render (`improbadoc serve <path-to-docs>`) to see the number rendering in your browser and check they are working correctly.

Reply:
Yes, this is 2 because it gets confused with the text code example above. I'll double check it looks fine in improbadoc though.